### PR TITLE
fix(core): fix multi-input bugs

### LINKED ISSUE #5789: multiinput bugs
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
- [ ] Multiinput - selecting a new item by clicking it anywhere other than the checkbox should close the popover (popover should stay open if checkbox is clicked)
ezgif-7-c3a3347f90cf

- [ ] Tokenizer (and multiinput) - cmd/ctrl+A to select all tokens then you must press delete twice to delete - should only happen once

- [ ] Multi-input should autocomplete like combobox

### PR DESCRIPTION
**Pull request by N1XUS**
#### Please provide a link to the associated issue.
Closes #5789

#### Please provide a brief summary of this pull request.
* Selecting a new item by clicking it anywhere other than the checkbox should close the popover (popover should stay open if the checkbox is clicked)
* cmd/ctrl+A to select all tokens then you must press delete twice to delete - should only happen once
* Multi-input should autocomplete like combobox
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- n/a update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 1]
```diff
@@ -421,24 +428,30 @@ export class MultiInputComponent implements
     }
 
     /** @hidden */
-    handleSelect(checked: any, value: any, event?: MouseEvent): void {
+    handleSelect(checked: any, value: any, resetSearch = true, event?: MouseEvent): void {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/multi-input/multi-input.component.ts:431]: missing prefix

File: libs/core/src/lib/token/tokenizer.component.ts [hunk 2]
```diff
@@ -402,6 +402,11 @@ export class TokenizerComponent implements AfterViewChecked, AfterViewInit, Afte
         this.moreClickedEvent.emit();
     }
 
+    removeSelectedTokens(): void {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/token/tokenizer.component.ts:405]: missing typedocs for the method

File: libs/core/src/lib/token/tokenizer.component.ts [hunk 3]
```diff
@@ -667,6 +672,10 @@ export class TokenizerComponent implements AfterViewChecked, AfterViewInit, Afte
         return document.activeElement === this.input.elementRef().nativeElement;
     }
 
+    private _tokensSelected(): boolean {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/token/tokenizer.component.ts:675]: typedocs missinf

File: apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-example/multi-input-example.component.html [hunk 4]
```diff
@@ -33,3 +33,17 @@
...
 Selected: [{{ thirdSelected.toLocaleString() }}]
+
+<br /><br />
+
+<label fd-label for="multiInputDisabledAutoComplete">Disable Auto Complete</label>
+<fd-multi-input
+    inputId="multiInputDisabledAutoComplete"
+    [autoComplete]="false"
+    [dropdownValues]="['Apple', 'Banana', 'Pineapple', 'Tomato', 'Kiwi', 'Strawberry', 'Blueberry', 'Orange']"
+    [placeholder]="'Search here...'"
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-example/multi-input-example.component.html:44]:
```suggestion
    placeholder="Search here..."
```

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**